tests/test_gcalcli.py

```diff
@@ -28,6 +65,52 @@ def default_options():
     return opts
 
 
+@pytest.fixture
```

Comment:
Essentially duplicating a fixture like this for the purpose of mocking `_search_for_events` method is not ideal but it is perhaps the start of some work on the test suite. I'm not an expert with `pytest` so I wasn't able to find a more elegant way of doing this at this stage, but it did allow me to test the CLI output - of which more like it is needed. I notice you have stubs for other tests - if we can find a better way to patch the GCalI object (your advice welcome), I'd be happy to do some work on this.